tests/README.md

```diff
@@ -6,33 +6,37 @@ quickly spin up a Kubernetes cluster. The Test framework is designed to install
 
 ## Install Kubernetes
 
-You can choose any Kubernetes install of your choice.  The test framework only depends on `kubectl` being configured.
-An optional helper script is provided to install Kubernetes with minikube.
+You can choose any Kubernetes install of your choice. The test framework only depends on `kubectl`
+being configured. To install `kubectl`, please see the [official guide](https://kubernetes.io/docs/tasks/tools/#kubectl).
 
 ### Minikube
 
+To install Minikube follow the [official
+guide](https://kubernetes.io/fr/docs/tasks/tools/install-minikube/). It is recommended to use the
```

Comment:
link to doc in French ... maybe is better to use:
https://kubernetes.io/docs/tasks/tools/#kubeadm

Reply:
😂  didn't even notice, will update!